labonneboite/alembic/sql/etablissements_initial.sql

```diff
@@ -67,7 +67,10 @@ SET @collation = (
 SET @encoding = (SELECT IFNULL(@user_favorite_offices_encoding, (SELECT IFNULL(@encoding, 'utf8'))));
 SET @collation = (SELECT IFNULL(@user_favorite_offices_collation, (SELECT IFNULL(@collation, 'utf8_general_ci'))));
 
-
+# WIP 2018.04 risky simplification : attempt at always starting directly with the right encoding and collation
+# FIXME delete and simplify old code only once absolutely sure no scenario fails
+SET @encoding = 'utf8mb4';
+SET @collation = 'utf8mb4_unicode_ci';
```

Comment:
Je croyais que ce fichier SQL était chargé avant d'appliquer les migrations, donc l'encoding et la collation étaient utf8 ?

Effetivement, ce changement est risqué parce qu'il impacte pas mal de migrations existantes...
Je pense que je serais rassuré au sujet de ce changement si les tests selenium de la branche regisb/selenium pasaient out of the box.

Reply:
il faut mettre fin à la prise de tête collation/encoding

pour s'en sortir le plus simple est que le projet oublie purement et simplement que dans le passé on avait une autre collation/encoding

démarrons partout de la bonne collation/encoding et effaçons cette dette technique

Reply:
si je me trompe pas, c'est similaire à du "squashing" de migrations... nettoyons le passé

Reply:
Si on fait du squashing, alors plutôt que de patcher les vieilles migrations, supprimons-les carrément toutes.